fractal_server/app/api/v1/workflow.py

```diff
@@ -52,3 +71,6 @@ async def get_list_workflow(
     List public workflows
     """
     raise NotImplementedError
+
+
+### ----------
```

Comment:
keep code clean: do not insert placeholder comments, as they rapidly make their way in the stable codebase, decreasing readability and overall code quality